ngraph/core/include/openvino/op/color_convert_nv12.hpp

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+#include "openvino/op/util/attr_types.hpp"
+
+namespace ov {
+namespace op {
+namespace v8 {
+/// \brief Color conversion operation from NV12 to RGB/BGR format.
+///    Input:
+///        - Operation expects input shape in NHWC layout.
```

Comment:
I think NV12 has well defined storage format for single and double plane cases, and I'm not sure how the transpose from `NV12 NHWC` to `NV12 NCHW` would look like. So my suggestion it to drop explicit layout specification for input tensor in this operation.